src/session.lisp

```diff
@@ -30,6 +30,9 @@
   (http-delete-check (session-path session "")))
 
 ;; TODO: make eldoc-friendly
+(defun use-session(session)
```

Comment:
Could you please exlain why this is useful?
We already have `start-interactive-session`

Reply:
start-interactive-session was much appreciated. The problem is when there are two parties involved. In my case - it is web chat application. With use-session it is possible to switch context and continue to use convenience to not specify active session
for example:
(defparameter *agent* (make-session))
(defparameter *visitor* (make-session))

(use-session *agent*)
(setf (url) "http://example.com/chatroom")
(element-send-keys (find-element "//input[@name='chat']" :by :xpath) "Hello  from agent")

(use-session *visitor*)
(setf (url) "http://example.com/chatroom")
(element-send-keys (find-element "//input[@name='chat']" :by :xpath) "Hello  from visitor")
(use-session *agent*)
(element-send-keys (find-element "//input[@name='chat']" :by :xpath) "reply from agent...")
...

Reply:
It seems that this scenario is quite seldom.
Probably this function should be defined in the user code but not in the public API. We can also add this code to the examples.

Reply:
I have tried to do that in user code, but *session* was not exported.
Options that I can think of are:
 - export *session*,
 - export a function that sets value for *session*, 
 - cl-selenium::*session* directly.
The last option seems hacky,
The first option - it could mess with the way *session* is handled - for example on initiate start-interactive-session - the last one  is closed, on stop - it is set to null
that is why, to me less obtrusive option is to use exported use-session.

Reply:
Makes sense.